File: src/Homework2/Tests/ProgramTests.cs
using Xunit;
using ConsoleApp1;

namespace TestProject1
{
    public class ProgramTests
    {
        [Fact]
        public void Main_WrongInput_WillReturnNotZero()
        {
            var args = new[] { "12", "+", "qu" };
            var result = Program.Main(args);
            Assert.True(result != 0);
        }
        
        [Fact]
        public void Main_CorrectInput_WillReturnZero()
        {
            var args = new[] { "12", "+", "34" };
            var result = Program.Main(args);
            Assert.True(result == 0);
        }
    }
}

File: src/Homework5/Homework5Tests/ProgramTests.fs
module Homework5Tests.ProgramTests

open Homework5
open Xunit
open Program
open Calculator

[<Fact>]
let Main_WrongInputData_WillReturnOne () =
    let args = [|"1984";"+";"qu"|]
    let check = main args
    let result = check = 1
    Assert.True(result)

[<Fact>]
let Main_CorrectInputData_WillReturnZero () =
    let args = [|"1984";"+";"34"|]
    let check = main args
    let result = check = 0
    Assert.True(result)
    
[<Fact>]
let Error_Division_By_Zero() =
    let args = [|"1984";"/";"0"|]
    let check = main args
    let result = check = 1
    Assert.True(result)